vpc/tool/assign3/assign_network.go

```diff
@@ -32,10 +32,12 @@ type Arguments struct {
 	Jumbo              bool
 	Bandwidth          uint64
 	Burst              bool
+	NetworkMode        string
```

Comment:
The titus executor and titus-vpc-tool are (effectively) atomically upgraded. So, you should get rid of the `AssignIPv6Address` logic here.

Reply:
I don't quite understand.
You want me to remove the `AssignIPv6Address` cli flag?

Reply:
Yeah, isn't `AssignIPv6Address` going to be subsumed by network mode?

Reply:
Eventually yes, but network mode will be Unknown (`0`) for a long time, we still need this bool to be in place to send in this data when network mode is 0.